extensions/amp-analytics/amp-analytics.md

```diff
@@ -59,6 +59,13 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
 </amp-analytics>
 ```
 
+#### <a name="attributes"></a>Attributes
+
```

Comment:
I left the text as it is. I think the whole doc could use some restructuring. Will you be able to take a stab at that?